src/ObjectEncryptor.php

```diff
@@ -52,7 +56,9 @@ public function encryptGeneric($data)
         return $this->encrypt(
             $data,
             $wrappers,
-            $this->encryptorOptions->getAkvUrl() ? GenericAKVWrapper::class : GenericKMSWrapper::class
+            $this->encryptorOptions->getAkvUrl() ? GenericAKVWrapper::class :
+                ($this->encryptorOptions->getGkmsKeyId() ? GenericGKMSWrapper::class :
+                    GenericKMSWrapper::class)
```

Comment:
Vnoreny ternary pls ne :slightly_smiling_face:

Reply:
To není vnořený ternary, jsou tam závorky, vnořený ternary je deprecated

Reply:
Kvaka to jako kachna, chodi to jako kachna, smrdi to jako kachna, ale ma to zavorky, tk to nebude kachna :rofl: 

Poradi vyhodnoceni je v pohode, ale kdo to ma cist.